api/tests/integration/application/schooling-registration-dependent-users/schooling-registration-dependent-user-controller_test.js

```diff
@@ -34,6 +34,8 @@ describe('Integration | Application | Schooling-registration-dependent-users | s
         'birthdate': '2012-12-12',
         'campaign-code': 'RESTRICTD',
         'password': 'P@ssw0rd',
+        'username': 'first.last1010',
```

Comment:
Histoire d'être plus près de l'algo en production et de mieux lire le test
```suggestion
        'username': 'Robert.Smith0001',
```

Reply:
Bien vu, par contre le pattern c'est prenom.nom[jour de naissance][mois de naissance] donc j'ai mis `Robert.Smith1212`